<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeManipulatorClass</class>
 <widget class="QMainWindow" name="NodeManipulatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NodeManipulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>573</width>
      <height>427</height>
     </rect>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::DragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::MoveAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="allColumnsShowFocus">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>574</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="FileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="ActionsMenu">
    <property name="title">
     <string>&amp;Action</string>
    </property>
    <addaction name="RemoveActionNode"/>
   </widget>
   <addaction name="FileMenu"/>
   <addaction name="ActionsMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>L&amp;oad</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>S&amp;ave</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Sa&amp;veAs</string>
   </property>
  </action>
  <action name="RemoveActionNode">
   <property name="text">
    <string>Remove Node</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="nodemanipulator.qrc"/>
 </resources>
 <connections/>
</ui>
